cumulusci/cli/cci.py

```diff
@@ -764,7 +764,10 @@ def service_info(config, service_name, plain):
 @click.argument("org_name", required=False)
 @pass_config
 def org_browser(config, org_name):
-    org_name, org_config = config.get_org(org_name)
+    try:
+        org_name, org_config = config.get_org(org_name)
+    except OrgNotFound:
+        raise click.ClickException(f"Org {org_name} does not exist in the keychain")
```

Comment:
I copied this error message from the `cci org remove` task, which already had this exception handling in place.